app/client/src/components/pages/MonitoringReport.js

```diff
@@ -250,12 +250,9 @@ const downloadLinksStyles = css`
 `;
 
 const fileLinkStyles = css`
-  background: none;
-  border: none;
+  ${iconButtonStyles}
   color: #0071bc;
-  margin: 0;
-  outline: inherit;
-  padding: 0;
+
```

Comment:
This is showing as black on hover and focus. For consistency, I think the following should be added:
```suggestion
&:hover,
&:focus, {
  color: ${colors.navyBlue()} !important;
}
```